contracts/libs/RollupUtils.sol

```diff
@@ -437,6 +450,14 @@ library RollupUtils {
         return Tx.serialize(_txs);
     }
 
+    function DecompressManyAirdrop(bytes memory txs)
+        public
+        pure
+        returns (Tx.Transfer[] memory structTxs)
+    {
+        return DecompressManyTransfer(txs);
```

Comment:
Reuse Transfer's for airdrop, because we use Tx.Transfer for them for the time being.